### PR TITLE
Docs/docker network spammer

### PR DESCRIPTION
**Pull request by piotrm50**
# Description of change

Added instruction on how to turn on spammer in local network. 

## Type of change

- Documentation Fix

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/tooling/docker_private_network.md [hunk 1]
```diff
@@ -67,3 +67,34 @@ csv
...
 ```
 Note, that the record length of the files might differ, since the approval check execution time of the nodes might differ.
+
+## Spammer tool
+
+Spammer tool lets you add messages to the tangle when running GoShimmer in a Docker network.
+In order to start spammer, you need to send GET request to a `/spammer` API endpoint with following parameters:
+* `cmd` - one of two possible values: `start` and `shutdown`.
+* `mpm` - messages per minute. Only applicable when `cmd=start`. 
+* `imif` - parameter indicating time interval between issued messages. Possible values
```
Comment (capossele) [inline docs/tooling/docker_private_network.md:77]: Maybe we should say that this parameter is optional?

File: docs/tooling/docker_private_network.md [hunk 2]
```diff
@@ -67,3 +67,34 @@ csv
...
+    * `exponential` -  emit messages by intervals whose random length is exponentially distributed
+    * `uniform` - issues messages at constant rate
+
+Example requests:
+
+```bash
+http://localhost:8080/spammer?cmd=start&mpm=1000&imif=uniform
```
Comment (capossele) [inline docs/tooling/docker_private_network.md:84]: Here I would provide an additional example without specifying imif

**Review by capossele — COMMENTED** [review]
🍻

File: docs/tooling/docker_private_network.md (hunk 2 above)
Comment (piotrm50) [reply docs/tooling/docker_private_network.md:86]: Done

File: docs/tooling/docker_private_network.md (hunk 1 above)
Comment (piotrm50) [reply docs/tooling/docker_private_network.md]: Done

**Review by capossele — APPROVED** [review]

**Review by apenzk — APPROVED** [review]

**Pull request merged**